src/components/app/content/cardList/card/index.js

```diff
@@ -32,21 +31,17 @@ function Card(props) {
 
     const fillData = (data) => setTempState({ ...tempState, ...data });
 
-    const selectCard = () => {
-        setEditable(false);
-        checkedControl.selectCard(cardId);
-    };
-    const editMode = () => {
-        setEditable(true);
-        checkedControl.removeCheckedCard(cardId);
-    };
+    const editMode = () => setEditable(true);
+
+    let history = useHistory();
+    const cardHandler = () =>
+        editable || history.push({ pathname: `/card/:${cardData.id}`, cardData });
```

Comment:
If try opening card with non-existing id - error is raised however app should be redirected to 404 page. Please fix.